tests/system/python/packages/test_eds.py

```diff
@@ -0,0 +1,186 @@
+# -*- coding: utf-8 -*-
+
+# FLEDGE_BEGIN
+# See: http://fledge.readthedocs.io/
+# FLEDGE_END
+
+""" Test EDS
+"""
+
+import os
+import subprocess
+import http.client
+import json
+import time
+import pytest
+from pathlib import Path
+import utils
+from datetime import datetime
+
+__author__ = "Yash Tatkondawar"
+__copyright__ = "Copyright (c) 2020 Dianomic Systems, Inc."
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+asset_name = "sinusoid"
+north_plugin = "OMF"
+task_name = "eds"
+# This  gives the path of directory where fledge is cloned. test_file < packages < python < system < tests < ROOT
+PROJECT_ROOT = Path(__file__).parent.parent.parent.parent.parent
+SCRIPTS_DIR_ROOT = "{}/tests/system/python/packages/data/".format(PROJECT_ROOT)
+FLEDGE_ROOT = os.environ.get('FLEDGE_ROOT')
+
+
+@pytest.fixture
+def reset_fledge(wait_time):
+    try:
+        subprocess.run(["cd {}/tests/system/python/scripts/package && ./reset"
+                       .format(PROJECT_ROOT)], shell=True, check=True)
+    except subprocess.CalledProcessError:
+        assert False, "reset package script failed!"
+
+    # Wait for fledge server to start
+    time.sleep(wait_time)
+
+
+@pytest.fixture
+def reset_eds():
+    eds_reset_url = "/api/v1/Administration/Storage/Reset"
+    con = http.client.HTTPConnection("localhost", 5590)
+    con.request("POST", eds_reset_url, "")
+    resp = con.getresponse()
```

Comment:
please print so that we can see it with -s -vv

Reply:
An HTTP status 204 message indicates success. verify that status code.
https://osisoft.github.io/Edge-Data-Store-Docs/V1/Administration/Administration_1-0.html